doc/configuration.rst

```diff
@@ -60,10 +60,21 @@ Top-level configuration
    :start-at: version:
    :end-before: # docs
 
-.. csv-table::
-   :header-rows: 1
-   :widths: 22,7,22,33
-   :file: configtables/toplevel.csv
+.. dropdown:: Details
+
+   .. jsonschema:: ../config/schema.json#/properties/version
+      :lift_description:
+
```

Comment:
This is the core information of the configuration page. I would not hide it in a dropdown, rather the other way around: the default config yaml could be in a dropdown.